designs/MPW_Design_Description.md

```diff
@@ -0,0 +1,851 @@
+
+-   `junga_soc_mpw5`
+    -   Project Name: junga_soc_mpw5
+    -   Project Owner: Lena Hwang
+    -   Project Number: [780](https://platform.efabless.com/projects/780)
+    -   Description: Simple vexriscv based SoC
+    -   Executable CI Script: [junga_soc_mpw5](../scripts/junga_soc_mpw5)
+
+-   `kasirga_k0`
+    -   Project Name: Kasirga K0
+    -   Project Owner: İsmail Emir Yüksel
+    -   Project Number: [638](https://platform.efabless.com/projects/638)
+    -   Description: RISC-V SoC
+    -   Executable CI Script: [c0_system_macro](../scripts/c0_system_macro)
+
+-   `keyvalue_caravel`
+    -   Project Name: Key Value store
+    -   Project Owner: Giray Pultar
+    -   Project Number: [509](https://platform.efabless.com/projects/509)
+    -   Description: A key value store using a wishbone interface,
+         developed using migen.
+    -   Executable CI Script: [keyvalue_caravel](../scripts/keyvalue_caravel)
+
+-   `koggestone_adder`
+    -   Project Name: caravel_koggestone_adder_project
+    -   Project Owner: Mohammed Zakir Hussain
+    -   Project Number: [640](https://platform.efabless.com/projects/640)
+    -   Description: 16-bit kogge-stone adder Verilog implementation.
+    -   In computing, the Kogge–Stone adder is a parallel prefix form
+         carry look-ahead adder.
+    -   Executable CI Script: [koggestone_adder](../scripts/koggestone_adder)
+
+-   `ks-guitar`
+    -   Project Name: Karplus-Strong Guitar
+    -   Project Owner: Tamas Hubai
+    -   Project Number: [648](https://platform.efabless.com/projects/648)
+    -   Description: Physically modeled guitar strings using the
+         Karplus-Strong algorithm with some extensions by Jaffe &
+         Smith.
+    -   Executable CI Script: [ks-guitar](../scripts/ks-guitar)
+
+-   `ks-guitar-2s`
+    -   Project Name: Karplus-Strong Guitar (two-strings)
+    -   Project Owner: Tamas Hubai
+    -   Project Number: [660](https://platform.efabless.com/projects/660)
+    -   Description: Two string version of Karplus-Strong Guitar
+    -   Executable CI Script: [ks-guitar-2s](../scripts/ks-guitar-2s)
+
+-   `logic_bist`
+    -   Project Name: Logic BIST
+    -   Project Owner: Dinesh Annaya
+    -   Project Number: [566](https://platform.efabless.com/projects/566)
+    -   Description: Logic BIST with Scan Chain to detect struck at
+         fault
+    -   Executable CI Script:
+
+-   `logic_bist`
+    -   Project Name: LBIST-MBIST
+    -   Project Owner: Dinesh Annaya
+    -   Project Number: [771](https://platform.efabless.com/projects/771)
+    -   Description: Logic built-in self-test (or LBIST) is a form of
+         built-in self-test (BIST) in which hardware and/or software is
+         built into integrated circuits allowing them to test their own
+         operation.
+    -   Executable CI Script:
+         [LBIST_mbist_top1](../scripts/LBIST_mbist_top1) |
+         [LBIST_mbist_top2](../scripts/LBIST_mbist_top2)
```

Comment:
Are these duplicated or is one missing from CI?

Reply:
Its not duplicate. Need to update 566 designs into CI later.

Reply:
consider its re-submitted design with improvement of designs from previous MPW shuttle to current one

Reply:
Its re-subitted MPW shuttle designs. Project Name and number will take to MPW shuttle path. But currently we are using latest designs in the CI